packages/yourloops/test/integration/assert/stats.ts

```diff
@@ -102,3 +103,8 @@ export const checkCoefficientOfVariationStatWidget = async (expectedTextContent:
   const patientStatistics = within(await screen.findByTestId('patient-statistics', {}, { timeout: 3000 }))
   expect(patientStatistics.getByTestId('coefficient-of-variation-stat')).toHaveTextContent(expectedTextContent)
 }
+
+export const checkTotalCarbsStatWidget = async (expectedTextContent: string) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I'm not convinced that this needs to be in a function

Reply:
Why? This function is called twice in trend-view and daily-view

Reply:
This function is really small and does not do much. Trying to make such small and simple stuff generic can end up making things more complex for little to no added value. In that case, I think that it was not worth it :slightly_smiling_face:

Reply:
yes sir